pymc_experimental/model_builder.py

```diff
@@ -74,8 +74,8 @@ def __init__(
         sampler_config = self.default_sampler_config if sampler_config is None else sampler_config
         self.sampler_config = sampler_config
         model_config = self.default_model_config if model_config is None else model_config
-
         self.model_config = model_config  # parameters for priors etc.
+        self.model_coords = None
```

Comment:
I think this property should be included in child classes IF they utilise coords. Otherwise it forces all classes that would inherit from MB even if they don't have Hierarchical nature.

Reply:
Thanks for your suggestion. The suggestion does not force any non-hierarchical model to populate the `coords` attribute. If a model has no coords, this can stay `None`